timepiece/reports/views.py

```diff
@@ -2241,12 +2241,11 @@ def get_context_data(self, **kwargs):
                 turn_in_date = milestones.get(name='Turn-In')
                 required_completion_date = milestones.get(
                     name='Required Completion')
-                required_time = (required_completion_date.due_date -
-                    start_date.due_date) + datetime.timedelta(days=1)
-                spent_time =  (turn_in_date.due_date -
-                    start_date.due_date) + datetime.timedelta(days=1)
+                required_time = (required_completion_date.due_date - 
+                                 start_date.due_date)
+                spent_time = (turn_in_date.due_date - start_date.due_date)
```

Comment:
Craig has indicated that a throughput of 0.0 is fine for projects that start and complete on the same day, and also add a day to the required_time if the completion date and start date are the same.